apps/server/src/shared/repo/filerecord/filerecord.repo.ts

```diff
@@ -11,17 +11,24 @@ export class FileRecordRepo {
 
 	async findOneById(id: EntityId): Promise<FileRecord> {
 		const fileRecord = await this.em.findOneOrFail(FileRecord, id);
+
 		return fileRecord;
 	}
+	// later findOneExpiresByIt
 
 	async save(fileRecord: FileRecord): Promise<void> {
 		await this.em.persistAndFlush(fileRecord);
 	}
 
+	async multiSave(fileRecords: FileRecord[]): Promise<void> {
```

Comment:
rename to saveMany

Reply:
Why we need that, we can use save(fileRecord: FileRecord | FileRecord[])